.github/workflows/runtests.yml

```diff
@@ -11,15 +11,15 @@ on:
 jobs:
   test:
     name: Julia ${{ matrix.version }} - ${{ matrix.tests }} - ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+    runs-on: self-hosted
```

Comment:
That's overkill you can add an extra to the matrix to do just one run on GPU

```
 matrix:
        version: ['1.6', '1.7', '1.8', '1.10']
         tests: ["basics", "layers", "networks"]
         os: [ubuntu-latest]

         include:
             - os: self-hosted
                tests:  whichever
                version: whichever
 ```
 
 and then just needs 
 ```
 runs-on: ${{ matrix.os }} 
 ```

Reply:
SMART thank you!